orchestration/orchestrate_unified.py

```
@@ -45,20 +36,20 @@ def _check_dependencies(self):
         for name, command in dependencies.items():
             try:
                 result = subprocess.run(
-                    ['which', command], 
-                    capture_output=True, 
+                    ['which', command],
+                    capture_output=True,
                     text=True
                 )
                 if result.returncode != 0:
                     missing.append(name)
             except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)